### ISSUE TITLE
Feedback Form

### ISSUE THREAD

**Issue by EmmanuelHere** [XollabOS/talktometechnology#49]
Can we create a feedback form to obtain input from those who utilize this app? This would allow for users to identify potential pain points so that we can continue to bettering this application!

**Comment by laurelayounis** [comment]
absolutely, I think the next step for the project is to create a user feedback form so we can built based off of user input, I'm going to keep this issue open until we have this done. Thank you so much for raising this issue.
